docs/sources/whatsnew/whats-new-in-v10-1.md

```diff
@@ -0,0 +1,496 @@
+
+{{< figure src="/media/docs/grafana/data-sources/tempo-search.png" max-width="750px" caption="Tempo Search editor powered by TraceQL" >}}
+
+### Span filtering for traces is GA
+
+_Generally available in all editions of Grafana_
+
+<!-- Joey Tawadrous -->
+
+Since the last release, we've made several improvements to span filtering. Now, we're promoting span filters out of public preview and into general availability.
+
+Span filters allow you to work much more efficiently with traces that consist of a large number of spans. Span filters exist above the trace view and allow you to filter the spans that are shown in the trace view. The more filters you add, the more specific the filtered spans.
+
+Currently, you can add one or more of the following filters:
+
+- Service Name
+- Span Name
+- Duration
+- Tags (which includes tags, process tags, and log fields)
+
+To only show the spans you've matched, you can enable the **Show matches only** toggle.
+
+Learn more about span filtering in our [Tempo data source documentation]({{< relref "../datasources/tempo/#span-filters" >}}).
+
+{{< figure src="/media/docs/tempo/screenshot-grafana-tempo-span-filters-v10-1.png" max-width="750px" caption="Traces span filtering" >}}
+
+### Configuration page redesign for Loki and Elasticsearch
+
+<!-- Matías Wenceslao Chomicki -->
+
+_Generally available in all editions of Grafana_
+
+The Loki and Elasticsearch data source configuration pages have been redesigned to make getting started and setting up data sources as simple and easy to understand as possible. You can now find new subsections with links to configuration pages, as well as tooltips to assist you with configuring and customizing data sources.
+
+### Loki query splitting
+
+<!-- Matías Wenceslao Chomicki -->
+
+_Generally available in all editions of Grafana_
+
+In response to different query performance scenarios, we've implemented query splitting. Now, queries that request more than a day of data are split into sub-requests, each with a duration of one day. For example, requesting seven days of logs will produce seven requests of one day.
+
+### Easier to use query editor for Elasticsearch
+
+<!-- Gabor Farkas -->
+
+_Generally available in all editions of Grafana_
+
+The Elasticsearch query editor now allows convenient switching between logs, metrics, and raw data directly from the top, eliminating the need to go through the metric selector.
+
+### Metrics explorer
+
+The Metrics explorer is a new feature that enhances metrics browsing in the Prometheus query builder. The Metrics explorer makes it easier for you to find the right metric, and get comfortable with PromQL. You can now explore metrics with additional metadata, perform fuzzy search on the metric name or description, and filter on the Prometheus type.
+
+{{< figure src="/media/docs/grafana/screenshot-grafana-10-1-metrics-explorer.png" max-width="750px" caption="Searching in Metrics explorer" >}}
+
```

Comment:
@iwysiu - Should the next three entries be under the Data sources section of the What's new?

**Note, please don't commit any changes without checking in with Quynton first.**

Reply:
Yes, they should be. Are they not?

Reply:
They are but the Redshift & Athena items are at the same level as the Data Sources heading, so it looks like they're their own sections. I'll fix the heading levels.